lisp/mastodon-tl.el

```diff
@@ -317,8 +320,7 @@ also render the html"
     (mastodon-tl--spoiler toot)
     ;; remove two trailing newlines
     (substring (mastodon-tl--content toot) 0 -2)
-    (when mastodon-tl--display-media-p
-      (mastodon-tl--media toot))
+    (mastodon-tl--media toot)
```

Comment:
We have only just made this change and now it is being undone again.  Would it make sense to just squash this commit with the first one?